### ISSUE TITLE
Examples: on CodeSandbox.io

### ISSUE THREAD

**Issue by fahad19** [Travix-International/frint#243]
Create some examples on https://codesandbox.io/, and see if we can even embed them in our docs. 

Making it easier for others to try things out.

**Comment by fahad19** [comment]
Two examples have been created so far:

* Basic rendering with React, and observing: https://codesandbox.io/s/6MNkoppL
* State management: https://codesandbox.io/s/j2VV1jWyY

**Comment by AlexDudar** [comment]
@fahad19 I see you made some examples on codesandbox, maybe we should create an `examples` page and display them there via codesandbox?

**Comment by fahad19** [comment]
Good idea. The examples page can mention the ones that exist in the monorepo, as well as the ones we host on codesandbox too.
